src/_data/companies.yml

```diff
@@ -1673,6 +1673,9 @@ Pharos Production Inc.:
   description: "Pharos Production team is made up of the brightest and most dedicated blockchain, fintech, ICO, web and mobile developers, designers, project managers and quality assurance specialists. Our development efforts resulted in more than 30 mobile and web solutions – social networking apps, ICOs, private blockchains, e-commerce, news boards, other different business and entertainment applications. We are hiring!"
   jobs:
     - title: Elixir Developer
+<<<<<<< HEAD
```

Comment:
@dmytronasyrov looks like you've got a merge conflict in this PR, could you rebase and remove these?